src/context/directory/handlers/pages.ts

```diff
@@ -60,24 +61,21 @@ function parse(context: DirectoryContext): ParsedPages {
 async function dump(context: DirectoryContext): Promise<void> {
   const pages = context.assets.pages;
 
-  if (!pages) return; // Skip, nothing to dump
+  if (!pages) return;
 
-  // Create Pages folder
   const pagesFolder = path.join(context.filePath, constants.PAGES_DIRECTORY);
   fs.ensureDirSync(pagesFolder);
 
   pages.forEach((page) => {
-    var metadata = { ...page };
+    const metadata = { ...page };
 
-    if (page.name !== 'error_page' || page.html !== undefined) {
-      // Dump template html to file
+    if (page.html !== undefined) {
```

Comment:
This is ultimately what fixes the issue. We don't care about the type of the page, but we do care if `page.html` is defined or not.